packages/kbn-apm-synthtrace/src/lib/apm/client/elastic_apm_http_client.d.ts

```diff
@@ -62,7 +62,7 @@ declare module 'elastic-apm-http-client' {
     /** If the service being instrumented is running a specific framework, use this config option to log its version */
     frameworkVersion?: string;
     /** Custom hostname (default: OS hostname) */
-    hostname?: string;
+    configuredHostname?: string;
```

Comment:
Yes, that looks correct. However, I wonder if this .d.ts file could be removed. See the comment above.

Reply:
Does that mean we can remove the direct dependency to the APM http client or do you need it elsewhere in Synthtrace?

Reply:
I can't see where it is used in synthtrace. I think the dep is no longer needed.

Reply:
I've removed the direct dependency to `elastic-apm-http-client` and this `.d.ts` file